perceval/runtime/rpc_handler.py

```diff
@@ -39,57 +41,94 @@
 
 
 class RPCHandler:
+    """Remote Call Procedure Handler
+
+    A class to call the API
+
+    """
+
     def __init__(self, name, url, token):
+        """Remote Call Procedure Handler
+
+        :param name: name of the plateform
+        :param url: api URL to call
+        :param token: token used for identification
+        """
         self.name = name
         self.url = url
         self.token = token
-        self.headers = {
-            'Authorization': f'Bearer {token}'
-        }
+        self.headers = {'Authorization': f'Bearer {token}'}
+        self.def_t = 10  # default timeout
```

Comment:
This variable name shouldn't be named `def_t`. Indeed, 10 is the default value for the request timeout, but the variable itself doesn't embody a default, it can be changed a the `RPCHandler` user.

I suggest you rename this variable `timeout` or `request_timeout`

Reply:
modified in my lastest commit https://github.com/Quandela/Perceval/pull/430/commits/e3816c9200156966a2d0454feb81e04ea5e37f6a